src/app/components/confirm/index.html

```diff
@@ -77,7 +77,7 @@
       propertyName="styleType"
       isOptional="true"
     >
-    A string that describes the button's visual appearance. <em>(Default: "default")</em>.
+    A string that describes the button's visual appearance. This property also accepts the <stache-code>primary</stache-code> and <stache-code>link</stache-code> values listed for <a stacheRouterLink="/components/button" fragment="classes">Button classes</a>. <em>(Default: "default")</em>.
```

Comment:
"also accepts" strikes me as weird since the default appears at the end. It's like we're alluding to something we haven't mentioned yet. Also, after looking at it a little more, I think "describes the button's visual appearance" is sorta vague when we can say that we are applying those SKY UX styles. What do you think of rewording the entire entry a bit to something like:
"styleType — (Optional.) A string that specifies a SKY UX button class to apply to the button. The valid options are `primary`, which formats the primary action on the dialog, `default`, which formats a secondary action, and `link`, which formats an action that takes users to another page."    (Note: I'm thinking "a SKY UX button class" would be the link to the button classes.)

Also, we don't include the default in the description. We set it through a property above. So you can add `defaultValue="default"` alongside the `propertyName` and isOptional` values in the lines above the description.